src/scripts/main.js

```diff
@@ -2,7 +2,36 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(person => {
+  person.sex = sexData(person.sex);
+  person.age = person.died - person.born;
+  person.century = centuryData(person.died);
+});
+
+function sexData(data) {
+  if (data === 'm') {
+    return 'Male';
+  } else {
+    return 'Female';
+  };
+}
+
+function centuryData(deathYear) {
+  return Math.ceil(deathYear / 100);
+}
+
+people.map(person => {
```

Comment:
In opposite, here you should use `forEach` method, as you are just doing some action for each person